www/src/components/guidelines/color/modal.js

```diff
@@ -17,12 +17,9 @@ const Column = ({ children, ...rest }) => (
     sx={{
       flexWrap: `wrap`,
       width: `100%`,
-      p: 6,
+      p: [6, null, null, 8],
```

Comment:
Hmmmm … what do you think about adding a constant for that?

Reply:
I never thought in that direction, will keep this also in mind. Thank you very much.

Reply:
🤗